README.md

```diff
@@ -7,10 +7,11 @@ Install dependencies:
 ```
     pip install beautifulsoup4
     pip install click
+    pip install pandas
```

Comment:
put all required libraries to the file.
create instruction about installing it from the file
- in other case you would need change docs each time you change ligraries
- further more it would make your future scripts more cleaner

in addition:

1. create instruction how to create venv
2. how to enter there
3. how to install all libraries there
4. how to run your script from it.